### DISCUSSION TITLE
Theme doesn't apply to profile page

### DISCUSSION THREAD

**Discussion by shawnpatel** [fief-dev/fief#300] [Q&A]
I'm trying to update the profile theme to dark mode, but some of the text isn't changing colors from the available theme color options. None of the theme colors seem to work. All the other pages have white text.
<img width="1399" alt="Screenshot 2023-11-20 at 8 55 12 PM" src="https://github.com/fief-dev/fief/assets/15237804/ac3332be-27c9-46fc-aa26-c5c07e64f7c2">

**Comment by fief-bailiff** [comment]
Hail, @shawnpatel 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
There are probably some miss in the variables assignment on the profile page. I'll file a bug about it!

**Reply by frankie567** [reply]
See #302

**Reply by shawnpatel** [reply]
Thanks!
